src/modules/real-time-metrics/filters/load-dataset-available-filters.js

```diff
@@ -67,22 +65,22 @@ const verifyWhitelistFields = ({ name, type: { name: typeName } }) => {
  * @return {Promise<Array<Object>>} An array of objects representing the available filters.
  */
 export default async function LoadDatasetAvailableFilters(dataset) {
-  if (cancelRequest) cancelRequest.cancel()
-  const ReportsRequestToken = Axios.CancelToken
-  cancelRequest = ReportsRequestToken.source()
+  if (abortController) abortController.abort()
```

Comment:
abortController is already defined as null, an always starts as null. I think we can change this.

Reply:
O controller é inicializado nulo propositalmente. Nas chamadas subsequentes, ele possui valor e por isso, o abort é chamado. Não tem motivo pra alterar esse comportamento em princípio.